aldjemy/orm.py

```diff
@@ -13,9 +13,8 @@
 
 def get_session(alias='default'):
     connection = connections[alias]
-    if not hasattr(connection, 'sa_session'):
-        session = orm.sessionmaker(bind=get_engine(alias))
-        connection.sa_session = session()
+    session = orm.sessionmaker(bind=get_engine(alias))
```

Comment:
why?

Reply:
I mean it is looks like a big deal. I'm not sure what consequences will be. So need to be sure what reason is.

Reply:
When models are mapped, the backref is taken from the related_name, and since the proxy model inherits all the fields from the parent, this results in duplicates of the backref and an error
"One or more mappers failed to initialize - can not proceed with initialization of other mappers." Triggering mapper: 'Mapper | Offer | core_offer'. Original exception was: Error creating backref 'advertiser_offers' on relationship 'Offer.advertiser': property of that name exists on mapper 'Mapper | User | core_user' "

Reply:
My question was about you delete `if`.

Reply:
Sorry, did not notice. The problem is in a closed connection to the database between requests.

InterfaceError: (psycopg2.InterfaceError) connection already closed

Only the first request is executed normally, then it tries to use the already closed connection.